### PR TITLE
feat: #85zrp3z63 - add browser tab name and icon, apply populated pat…

### PR DESCRIPTION
**Pull request by mal2tin**
…tern

# 🤘 Flutter PRs

| :point_up: don't even dare to be here if you didn't check [before_pr.md](./docs/before_pr.md) file |
| --- |

Thank you for adding your efforts to this awesome project and Flutter Team! 🤘
Please, keep your PRs short to make everybody happy

_Made with love by the_ **Monkslab** ❤️

## Keep it clean

These are our selected best practices that we agree to follow for any PR to be approved. We think it's super handy for including everything without having to think about it. We hope they will make your life easier! 🙏

## Type

- [x] New feature
- [ ] Testing
- [ ] Refactor
- [ ] Bug-fix

## Versions

- [x] upgrade with [Semantic Versioning](https://semver.org/) in pubspec.yaml: **latest here**

## Screenshots

https://user-images.githubusercontent.com/21958494/221959367-4ad4ba30-f0d6-40b0-8af4-2beedd2fb734.mov

### CODE REVIEW

File: lib/src/features/archive/presentation/widgets/archive_populated.dart [hunk 1]
```diff
@@ -17,51 +18,62 @@ class ArchivePopulated extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setApplicationSwitcherDescription(
```
Comment (maurodibert) [inline lib/src/features/archive/presentation/widgets/archive_populated.dart:21]: Does this also work with other browsers?

File: lib/src/features/article/article_populated.dart [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:the_monkslab_web/src/features/article/_index.dart';
+import 'package:the_monkslab_web/src/features/article/presentation/article_footer.dart';
+import 'package:the_monkslab_web/src/models/_index.dart';
+import 'package:the_monkslab_web/src/ui/_index.dart';
+import 'package:the_monkslab_web/src/utils/_index.dart';
+
+class ArticlePopulated extends StatelessWidget {
```
Comment (maurodibert) [inline lib/src/features/article/article_populated.dart:9]: Totally in loved ❤️

File: lib/src/features/article/presentation/view.dart [hunk 3]
```diff
@@ -60,69 +58,18 @@ class _ArticleViewState extends State<ArticleView> {
...
-                        child: ArticleBody(data: article.content),
-                      ),
-                      AppGaps.gapH48,
-                      const ArticleFooter(),
-                    ],
-                  ),
-                ),
-              );
-            }
+            return ArticlePopulated(
```
Comment (maurodibert) [inline lib/src/features/article/presentation/view.dart:68]: Even more in loved... 🤤

File: lib/src/features/course/presentation/course_populated.dart [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:the_monkslab_web/src/features/section/_index.dart';
+import 'package:the_monkslab_web/src/models/_index.dart';
+import 'package:the_monkslab_web/src/ui/_index.dart';
+
+class CoursePopulated extends StatelessWidget {
```
Comment (maurodibert) [inline lib/src/features/course/presentation/course_populated.dart:7]: Well... I should start stop doing this...

File: lib/src/features/course/presentation/view.dart [hunk 5]
```diff
@@ -34,32 +34,9 @@ class CourseView extends StatelessWidget {
...
-                      style: AppTextStyles.p,
-                    ),
-                  ),
-                  AppGaps.gapH24,
-                  for (var section in state.course!.sections)
-                    SectionPage(path: section.path)
-                ],
-              ),
-            ),);
+            return CoursePopulated(
```
Comment (maurodibert) [inline lib/src/features/course/presentation/view.dart:37]: ...but it doesn't hurt anybode! 🤣

File: web/index.html [hunk 6]
```diff
@@ -15,10 +16,9 @@
     the `--base-href` argument provided to `flutter build`.
   -->
   <base href="$FLUTTER_BASE_HREF">
-
   <meta charset="UTF-8">
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
-  <meta name="description" content="A new Flutter project.">
+  <meta name="description" content="Learn to cose as we had Dreamed.">
```
Comment (maurodibert) [inline web/index.html:21]: typo: co'd'e

**Review by maurodibert — DISMISSED** [review]
Super nice bro! Just one typo in the 'js' file. I've blocked this because I don't know where that typo could be showed!

Cool! ⚡️

File: web/index.html (hunk 6 above)
Comment (mal2tin) [reply web/index.html:21]: done!

File: lib/src/features/course/presentation/course_populated.dart (hunk 4 above)
Comment (mal2tin) [reply lib/src/features/course/presentation/course_populated.dart:7]: hehe

File: lib/src/features/archive/presentation/widgets/archive_populated.dart (hunk 1 above)
Comment (mal2tin) [reply lib/src/features/archive/presentation/widgets/archive_populated.dart:21]: yes, atleast with all who has based on chrome. I will try it with safari.

**Review by maurodibert — APPROVED** [review]
Cool! ❤️

**Pull request merged**